apps/test/unit/templates/manageStudents/MoveStudentsTest.js

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 import {expect} from '../../../util/configuredChai';
-import MoveStudents from '@cdo/apps/templates/manageStudents/MoveStudents';
+import {UnconnectedMoveStudents as MoveStudents, DEFAULT_STATE} from '@cdo/apps/templates/manageStudents/MoveStudents';
 
 const studentData = [
   {id: 1, name: 'studentb'},
   {id: 3, name: 'studenta'},
   {id: 0, name: ''},
   {id: 2, name: 'studentf'}
 ];
+const sections = [
+  {id: 0, name: 'sectiona'},
+  {id: 1, name: 'sectionb'},
+  {id: 2, name: 'sectionc'}
+];
```

Comment:
This sections array is basically the same as the one used in the Story. If there are other similar constants used, we could consider extracting to their own file.

Reply:
I'd say you'll get an A+ for extracting a shared constant here, but I'll also add that we're generally much more lenient about allowing duplication in our stories and tests than we are in the rest of our codebase. in most tests and stories there would be no real danger for this data to change in one place but not the other, and I might only consider extracting test data if it becomes quite complex or if the format is changing a lot. For example we do have a [generateFakeProjects](https://github.com/code-dot-org/code-dot-org/blob/5f0f6b6bbdf97a8f11c434f0f8a7b643a580b22a/apps/test/unit/templates/projects/ProjectAppTypeAreaTest.js#L10) method which is very useful because it produces fairly complex test data.

Reply:
Apologies, @Erin007 , as I've sat quiet as these kind of extractions have been suggested in the past 😊